frontend/src/components/QRCodeCard.tsx

```diff
@@ -90,7 +96,7 @@ export default function QRCodeCard({
           includeMargin={true}
         />
         <Typography textAlign='center' variant='body1' gutterBottom>
-          Tank {qrSymbol}
+          {nickname} {qrSymbol}
```

Comment:
so it'll say 

`Seahorse 3`, for example? I tthink the idea was doing away with the '3' part of the labelling altogether.

Reply:
left it there for the edge case that one customer has two(or more) tanks named 'Seahorse'. We don't have a good way of dealing with this really, and it keeps coming back to not having a solid unique identifier for a tank

Reply:
I believe we still want both. Nickname is like the description, and qrSymbol is like the index for each customer. (1-20)

So it's like saying "saltwater betta fish, tank #3"  <descriptor (nickname), index> rather than replacing the index with discrete strings like "clownfish, coral,..." and potentially causing confusion if we have "clownfish" as the index, but it's a freshwater tank that doesn't have any.